### ISSUE TITLE
Add pagination to transaction lists

### ISSUE THREAD

**Issue by skuro** [d-cent/freecoin#8]
Currently transactions (either user-specific or system-wide) are listed without any pagination, which is not suitable for a production deployment.

**Comment by jaromil** [comment]
True. We need pagination that doesn't requires javascript. Scrolling pagination may be optional and I'd rather leave it for later. Since there are other visualisations that also need pagination, I wonder how to best make a generic function for that in views.

**Comment by skuro** [comment]
Pagination could possibly be done at the storage level, as otherwise all list queries would result in the whole collections being passed around. In this scenario, we could define a standard set of query parameters (e.g. `?page=X&skip=Y&pageSize=Z`), add a `:pagination` entry in the liberator context and let lower levels take care of that. 
Responsibility of the view would always be to render the pagination links, for which both the full size of the queried list and the pagination query parameters will be used.
Javascript is not really needed here, and all we can build in the above way can also be completely reused when we decide to offer a "pure" API, with `application/json` being offered instead of `text/html`.

**Comment by jaromil** [comment]
Sounds great. Other use cases of pagination: search results and participants lists.
